app/models/submission.rb

```diff
@@ -190,6 +190,51 @@ def check_file_properties(metadata, sort)
     { sort => errors }
   end
 
+  def self.bulk_corrections!(tutorial, assignment, files)
```

Comment:
This method replaces Tutorial#add_bulk_corrections! which hence becomes redundant, but it is still contained in tutorial.rb.